src/panels/config/lovelace/resources/dialog-lovelace-resource-detail.ts

```diff
@@ -214,6 +215,15 @@ export class DialogLovelaceResourceDetail extends LitElement {
     this._params = undefined;
   }
 
+  private _detectResourceType(url: string) {
+    const ext = url.split(".").pop() || "";
+    const styleSheetExt = ["css", "scss", "sass"];
```

Comment:
This should be at the top of the file

Reply:
You mean outside of the class as a static global function?

Reply:
Correct

Reply:
This hasnt been addressed

Reply:
Did I miss something? The function is no longer part of the class since my latest commit from 4 hours ago.

Reply:
oh
I was only talking about the const var of the css types. That should be outside of the function. This is good too. But still no need to create the car everytime